build.sbt

```diff
@@ -54,13 +54,13 @@ val commonSettings = Seq(
 
 lazy val dockerSettings = Seq(
   packageName in Docker := "kafka-message-scheduler",
-  dockerBaseImage := "openjdk:8u171-jre-alpine",
+  dockerBaseImage := "alpine:3.13.2",
```

Comment:
We should use the image from our artifactory: `artifacts.ath.bskyb.com:5003/alpine:3.13.2`